samples/teams-blazor-server/Pages/Index.razor

```diff
@@ -0,0 +1,30 @@
+@page "/"
+
+
+
+<h2>Hello, MGT world!</h2>
+
+<p>
+    Welcome to your new app. It uses the Microsoft Graph Toolkit to log you in and shows details for your user.
+    <br />
+    Click on the sign in button to login. The very first time the pop up will ask you for consent for your Azure AD app.
+</p>
+
+<div>
+    <mgt-teams-provider 
```

Comment:
Consider adding the msal provider here as well. It will make the sample work in the browser and Teams and it will just make it easier to run. 

This code will allow the sample to use the msal provider when running in a browser and the teams provider when running in teams (the `depends-on` is key here) :
```
    <mgt-msal-provider
      client-id="a974dfa0-9f57-49b9-95db-90f04ce2111a"
      scopes="user.read,people.read,user.readbasic.all,contacts.read,calendars.read"
      depends-on="mgt-teams-provider"
      login-type="popup"
    ></mgt-msal-provider>
```

You can then rename this sample to simply "Blazor sample"